tests/integration-tests/tests/storage/test_efs/test_efs_compute_az/pcluster.config.yaml

```diff
@@ -38,7 +38,8 @@ Scheduling:
           MaxCount: 1
       Networking:
         SubnetIds:
-          - {{ private_subnet_ids[2] }}
+          # us-isob-east-1 provides 2 availability zones.
+          - {% if region == "us-isob-east-1" %} {{ private_subnet_ids[1] }} {% else %} {{ private_subnet_ids[2] }} {% endif %}
```

Comment:
After the multi AZ support, we could merge test_efs_compute_az and test_efs_same_az. 
One of the queue should be in `private_subnet_ids[0]`. The other queue should be in `private_subnet_ids[1]`.
The benefits are:
1. The test will only require 2 AZs in the region
2. We reduce a redundant test
3. We don't have a special condition for the isolated region

That said, if you would merge this PR first, I am fine